libr/egg/p/sc/asm/arm-linux-binsh.rasm

```diff
@@ -0,0 +1,22 @@
+; ARM 32-bit Linux execve /bin/sh shellcode
+; Assembly equivalent of libr/egg/p/sc/src/arm-linux-binsh.c
+;
+; Execve("/bin/sh", ["/bin/sh", NULL], NULL) using syscall 0xb
+
```

Comment:
use .arch and .bits directives to make this .rasm file assemble fine with rasm2.